### ISSUE TITLE
Union bitwise operator error on Python 3.9

### ISSUE THREAD

**Issue by sahiljhawar** [swansonk14/p_tqdm#49]
I am not sure how the CI build passed on GitHub Python 3.9 but it is not working when I am trying to install `p_tqdm` from source on Python 3.9, it is also not getting installed from PyPi. However it is getting installed from Conda, I am not sure why. The following error occurs while installing with `pip`:

```bash
__version__: str | None = None
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```

Maybe use typing for `__version__`

**Comment by sahiljhawar** [comment]
We have already pushed our package (`NMMA`) to PyPi a few days ago without pinning `p_tqdm`. Would it be possible for you to yank the current version and push again with the fix?

**Comment by swansonk14** [comment]
Hi @sahiljhawar,

Thank you for bringing up this issue and sorry for the error! I'm so used to working with `|` as the Union operator in newer versions of Python that I forgot that it would cause issues for Python <3.10. I merged in your pull request with the fix, pushed a new version (1.4.2), and yanked the broken version (1.4.1).

Thanks again for your help and please let me know if you run into any more issues!

Best,
Kyle

**Comment by sahiljhawar** [comment]
thanks @swansonk14!
